api/v1/rpc_clusterapi_certificates_plan.go

```diff
@@ -0,0 +1,9 @@
+package apiv1
+
+const ClusterAPICertificatesPlanRPC = "x/capi/refresh-certs/plan"
+
+// ClusterAPICertificatesPlanRequest is the request message for the ClusterAPICertificatesPlan RPC (same as RefreshCertificatesPlan).
+type ClusterAPICertificatesPlanRequest RefreshCertificatesPlanRequest
+
+// ClusterAPICertificatesPlanResponse is the response message for the ClusterAPICertificatesPlanRPC (same as RefreshCertificatesPlan).
```

Comment:
I feel guilty to say this but maybe there's a space missing in `ClusterAPICertificatesPlanRPC` or there's an additional space in the comment of the previous type (`ClusterAPICertificatesPlan RPC`).